src/Cameras/VR/babylon.vrExperienceHelper.ts

```diff
@@ -414,7 +419,7 @@ module BABYLON {
                         && mesh.name.indexOf("teleportationCircle") === -1
                         && mesh.name.indexOf("torusTeleportation") === -1
                         && mesh.name.indexOf("laserPointer") === -1)) {
-                    return true;
+                    return (mesh.isEnabled() && mesh.isVisible && mesh.isPickable);

```

Comment:
is this predicate configurable somehow?

Reply:
No, see commit message - basically I think here instead of "return true" we should be calling a configurable predicate - possibly a new one, but was thinking the existing one.  The intention of two predicates appears to be to ignore specific experience helper meshes.
```javascript
public meshSelectionPredicate
```
Calling the existing predicate would affect _castRayAndSelectObject function.

The current predicate is being used *after* the pickWithRay, so return true is not workable as we cannot pick beyond an unpickable mesh.

I think another predicate change to be made as well in case floorMeshName is substring of another mesh:
```javascript
mesh.name === this._floorMeshName
```

Reply:
Like the idea of a cutom predicate could you make it part of the option to allow override the default behaavior ?